pyogrio/tests/test_arrow.py

```diff
@@ -304,3 +310,270 @@ def test_arrow_bool_exception(tmpdir, ext):
     else:
         with open_arrow(filename):
             pass
+
+
+# Point(0, 0)
```

Comment:
This also gets used in a variety of places, perhaps this could be added as a function in `conftest.py` and then just call it when used below?

```Python
def get_wkb_points(count=1):
    return np.array([bytes.fromhex("010100000000000000000000000000000000000000")]*count, dtype=object)
```

We could hook it up to other usages in a different PR, this would just set us up for that while simplifying here.